docs/source/models/field_ionization_comparison_c_ii_ionization.py

```diff
@@ -97,7 +97,7 @@ def ADK_rate_simple(Z, E_i, F):
     plt.xlim([1e-2, 1e1])
     plt.yscale("log")
     plt.xscale("log")
-    plt.ylabel("ionization rate $\Gamma$ [Hz]")
+    plt.ylabel("ionization rate $\Gamma$ [\mathrm{s}^{-1}]")
```

Comment:
nooooo, this is NOT in math `$$` and I just merged it. pls fix in a follow-up to `[s$^{-1}$]`